repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/cta-card.ftl

```diff
@@ -14,6 +14,10 @@
         <#if link?has_content>
             <div class="nhsuk-grid-column-one-third">
                 <div class="nhsuk-card nhsuk-card--clickable">
+                    <@hst.link var="cardImage" hippobean=ctaCard.ctaCardContentBlock.cardImage />
+                    <#if cardImage??>
+                        <img class="nhsuk-card__img" src="${cardImage}" alt="">

```

Comment:
Could we render `cardImage` description as alt text ?

Reply:
done.